scripts/host-check

```diff
@@ -126,6 +126,22 @@ def purple(string: str) -> str:
 _CMD = Union[str, List[str]]
 
 
+class CmdTimeoutError(Exception):
+    """
+    Error to "replace" subprocess.TimeoutExpired
+    This is so that the error is treated differently from all the other errors
```

Comment:
"This is so that command timeouts can be handled separately to other subprocess.SubprocessError errors."